src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/Address.kt

```diff
@@ -3,7 +3,29 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Address as IntegrationAPIAddress
 
 data class Address(
-  val postalCode: String,
+  val addressType: String?,
+  val country: String?,
+  val county: String?,
+  val endDate: String?,
+  val flat: String?,
+  val locality: String?,
+  val postalCode: String?,
+  val premise: String?,
+  val startDate: String?,
+  val street: String?,
+  val town: String?,
```

Comment:
Can we reorder them?
  val flat: String?,
  val premise: String?,
  val street: String?,
  val town: String?,
  val locality: String?,
  val county: String?,
  val country: String?,
  val postalCode: String,
  val addressType: String?,
  val startDate: String?,
  val endDate: String?,